pages/index.tsx

```diff
@@ -2,7 +2,8 @@ import type { NextPage } from 'next'
 import { Box, Container, Heading } from '@chakra-ui/react'
 import { useAccount, useConnect, useEnsName } from 'wagmi'
 import { InjectedConnector } from 'wagmi/connectors/injected'
-import { useMounted } from '@/hooks'
+import { Approve } from 'components/metaMask/Approve'
```

Comment:
What do you think about renaming this to `components/Approve/index.tsx`?

I think it is more blockchain related than it is MetaMask related.

Reply:
I agree. Fixed.